### PR TITLE
Add French rss feed

### PR DESCRIPTION
**Pull request by walshbr**
Adds the French translations to the RSS feed.

### CODE REVIEW

**Comment by walshbr** [comment]
Sorry - the branch had gone stale w/ changes to gh-pages. Just re-merged gh-pages back into this branch. That should fix the travis errors.

**Comment by walshbr** [comment]
Closes #1414

**Comment by walshbr** [comment]
Doesn't require any translation, so it should be ready to merge pending review from someone on the tech team.

**Review by mdlincoln — APPROVED** [review]

**Pull request merged**
